_posts/2024-12-09-this-month-in-servo.md

```diff
@@ -0,0 +1,174 @@
+---
+layout:     post
+tags:       blog
+title:      "This month in Servo: :is() and :where(), grid layout, parallel flexbox, and more!"
+date:       2024-12-09
+summary:    Plus shadow DOM buffs, smaller binaries, and new SubtleCrypto algorithms (HMAC, AES-GCM, AES-KW, PBKDF2).
+categories:
+---
+
+<figure class="_figr"><a href="{{ '/img/blog/november-2024.png' | url }}"><img src="{{ '/img/blog/november-2024.png' | url }}" style="width: 33em;"
+    alt="Servo nightly showing new support for CSS grid layout"></a></figure>
+
+Servo now supports **CSS grid layout** (@nicoburns, @taniishkaa, #32619, #34352, #34421), **parallel layout for flexbox** (@mrobinson, #34132), and **:is() and :where() selectors** (@mrobinson, #34066)!
```

Comment:
Should we mention grid layout is not enabled by default and that you need to add a flag.

Reply:
done in a4e23e62454a8762be1bb79a0b550d06a2d2d6b5

Reply:
done in 15436f368661f0c58f65c7e86e16dff6e9cb1267